src/js/bs3/settings.js

```diff
@@ -188,6 +188,20 @@ $.summernote = $.extend($.summernote, {
       lineNumbers: true
     },
 
+    codeviewFilter: false,
+    codeviewRegex: /<\/*(?:applet|b(?:ase|gsound|link)|embed|frame(?:set)?|ilayer|l(?:ayer|ink)|meta|object|s(?:cript|tyle)|t(?:itle|extarea)|xml)[^>]*?>/gi,
+    codeviewIframeWhitelistSrc: [],
+    codeviewIframeWhitelistSrcOrigin: [
+      'www.youtube(?:-nocookie)?.com',
+      'www.facebook.com',
+      'vine.co',
+      'instagram.com',
+      'player.vimeo.com',
+      'www.dailymotion.com',
+      'player.youku.com',
+      'v.qq.com'
```

Comment:
Could you add a trailing comma here and also for other settings?
We recently changed our lint rule to add trailing commas for multiline.
https://github.com/summernote/summernote/pull/3055

Reply:
thank you for your feedback
i will commit it on next week